### PR TITLE
Stub error strings

### PR DESCRIPTION
**Pull request by mvidner**
AKA ["a 30 minute hack week project"](https://hackweek.suse.com/12/projects/166).

### CODE REVIEW

File: package/libstorage.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May 18 15:01:44 UTC 2015 - mvidner@suse.com
+
+- Added stub error strings. Now you should see
+  "DISK_PARTITION_NOT_FOUND" instead of "-1008".
```
Comment (aschnell) [inline package/libstorage.changes]: Actually in addition to the number, but that depends on the user of the library.

File: storage/Storage.cc [hunk 2]
```diff
@@ -5053,8 +5053,539 @@ namespace storage
     {
 	switch (error)
 	{
+	    // When replacing stub texts
+	    // remember to add translation markers too:
+	    //
+	    // return _("Volume too loud.").text
```
Comment (aschnell) [inline storage/Storage.cc]: Missing ";". You use to much Ruby.

**Comment by aschnell** [comment]
LGTM

**Comment by kobliha** [comment]
Thank you! :))))

**Pull request merged**